vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -9,32 +10,89 @@ import hudson.AbortException
-    'containerShell', // allows to specify the shell to be executed for container with containerName
-    'containerWorkspaces', //specify workspace (=home directory of user) per container. If not provided dockerWorkspace will be used. If empty, home directory will not be set.
+    /**
+     * Allows to specify start command for container created with dockerImage parameter to overwrite Piper default (`/usr/bin/tail -f /dev/null`).
+     */
+    'containerCommand',
+    /**
+     * Specifies start command for containers to overwrite Piper default (`/usr/bin/tail -f /dev/null`).
+     * If container's defaultstart command should be used provide empty string like: `['selenium/standalone-chrome': '']`.
+     */
+    'containerCommands',
+    /**
+     * Specifies environment variables per container. If not provided `dockerEnvVars` will be used.
+     */
+    'containerEnvVars',
+    /**
+     * A map of docker image to the name of the container. The pod will be created with all the images from this map and they are labled based on the value field of each map entry.
+     * Example: `['maven:3.5-jdk-8-alpine': 'mavenExecute', 'selenium/standalone-chrome': 'selenium', 'famiko/jmeter-base': 'checkJMeter', 's4sdk/docker-cf-cli': 'cloudfoundry']`
+     */
+    'containerMap',
+    /**
+     * Optional configuration in combination with containerMap to define the container where the commands should be executed in.
+     */
+    'containerName',
+    /**
+     * Map which defines per docker image the port mappings, e.g. `containerPortMappings: ['selenium/standalone-chrome': [[name: 'selPort', containerPort: 4444, hostPort: 4444]]]`.
+     */
+    'containerPortMappings',
+    /**
+     * Specifies the pullImage flag per container.
+     */
+    'containerPullImageFlags',
+    /**
+     * Allows to specify the shell to be executed for container with containerName.
+     */
+    'containerShell',
+    /**
+     * Specifies a dedicated user home directory per container which will be passed as value for environment variable `HOME`. If not provided `dockerWorkspace` will be used.
+     */
+    'containerWorkspaces',
+    /**
+     * Environment variables to set in the container, e.g. [http_proxy:'proxy:8080'].
+     */
+    'dockerEnvVars',
+    /**
+     * Name of the docker image that should be used. If empty, Docker is not used.
+     */
     'dockerImage',
+    /**
+     * Set this to 'false' to bypass a docker image pull.
+     * Usefull during development process. Allows testing of images which are available in the local registry only.
+     */
     'dockerPullImage',
+    /**
+     * Specifies a dedicated user home directory for the container which will be passed as value for environment variable `HOME`.
+     */
     'dockerWorkspace',
-    'dockerEnvVars',
```

Comment:
unnecessary move of parameter

Reply:
I corrected the order to match with the generated one.

Reply:
understood